src/pkg/db/utils.go

```diff
@@ -38,6 +38,9 @@ type DB interface {
 
 	// FindOutputMappingsByModuleID DEPRECATED fetch the terraform module along with it's attribute and output mappings of the attribute.
 	FindOutputMappingsByModuleID(ids ...uuid.UUID) (result TFModules, err error)
+
+	FetchDependencyByInterfaceID(interfaceID string, filterOps ...FilterOption) (result Dependencies, err error)
+	FetchAllDependency(filterOps ...FilterOption) (result Dependencies, err error)
```

Comment:
according to the naming convention set above, `FetchAllDependency` should be renamed to `QueryDependencies`

Reply:
Done